TLM/TLM/UI/UIBase.cs

```diff
@@ -165,14 +165,15 @@ public class UIBase : UICustomControl {
 
         public static void DisableTool() {
             Log._Debug("LoadingExtension.DisableTool: called");
-            if (ToolsModifierControl.toolController != null) {
+            if (ToolsModifierControl.toolController == null) {
```

Comment:
Add comment here that it's likely due to hotloading? Otherwise, anyone looking at this in future might struggle to see the connection.

Reply:
This code was a bit clumsy on my side. It is now fixed.
Its not only for hot reload though ... see description:
``` prevent unintentional disabling of other tools.```
Does not make sense to disable tool if TMPE is not the current tool.